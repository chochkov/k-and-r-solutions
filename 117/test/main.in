my name is ivan
and my line is too long
so it won't be printed
my name is ivan and my line is too long so it won't be printed
