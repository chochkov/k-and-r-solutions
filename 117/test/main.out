and my line is too long
my name is ivan and my line is too long so it won't be printed
